src/api/v2/player.py

```diff
@@ -13,7 +13,7 @@
 
 @router.get("/players/score", response_model=list[KCResponse])
 async def get_players_kc(
-    name: Annotated[list[str], Query(..., max_length=13)], session=Depends(get_session)
+    name: Annotated[list[str], Query(...)], session=Depends(get_session)

```

Comment:
length is not the length of the string, it's the length of the list. so it would limit how many players to send.  would have to integrate what I did in feedback if you want to add a str length limit

Reply:
oh interesting good catch! yeah we need to chech the lenght of each str, player names equal or longer than 13 char are not possible

Reply:
will add this 
```python
    name: list[Annotated[str, Field(..., min_length=1, max_length=13)]] = Query(
        ...,
        min_length=1,
        description="Name of the player",
        example=["Player1", "Player2"],
    )
```

Reply:
[str between 1-13 and at least 1 str, player/report/score](https://github.com/Bot-detector/public-api/pull/25/commits/1cf961c360e63de882247f3357a5f881cc2684ea)